packages/sources/stader-labs/src/endpoint/sftmx.ts

```diff
@@ -26,6 +26,7 @@ export const execute: ExecuteWithConfig<Config> = async (
   const jobRunID = validator.validated.id
   const provider = new ethers.providers.JsonRpcProvider(
     config.adapterSpecificParams?.fantomRpcUrl.toString(),
```

Comment:
why toString?

Reply:
Haven't touch this part but i see that adapterSpecificParams have type [T: string]: string | number so this is just for types